imapclient/imapclient.py

```diff
@@ -312,7 +312,7 @@ def capabilities(self):
         # If the server returned an untagged CAPABILITY response
         # (during authentication), cache it and return that.
         untagged = _dict_bytes_normaliser(self._imap.untagged_responses)
-        response = untagged.pop(b'CAPABILITY', None)
+        response = untagged.pop('CAPABILITY', None)
```

Comment:
It was an attempt to fix a BytesWarning exception, that you harvest from a IMAPClient.capabilities() call with Python 3, running with -bb.

Now, that I look, is seems strange, because it was going though the _dict_bytes_normailizer before. Here's the traceback:
```
  File "imap4.py", line 62, in backup
    self.imap.capabilities()))
  File "imapclient/imapclient.py", line 315, in capabilities
    response = untagged.pop(b'CAPABILITY', None)
  File "imapclient/imapclient.py", line 1439, in pop
    return self._d.pop(k)
BytesWarning: Comparison between bytes and string
```
Okay, not that surprising, given, the keys yielded from _gen_keys may be both, strings and bytes. These are the times, where I regret to have been moved to python3..